AR-Top-MobileApp/Assets/VoxelFinder.cs

```diff
@@ -29,9 +29,15 @@ public class VoxelFinder : MonoBehaviour
 				Vector3 offset = mapController.getOffset () * 10;
 				print (offset); 
 				Vector3 hitObjectPos = hitObject.transform.position * 10;
-				coordinatesText = "(" + Math.Round (hitObjectPos.x - offset.x) + ',';
-				coordinatesText += Math.Round (hitObjectPos.z - offset.z) + ",";
-				coordinatesText += Math.Round (hitObjectPos.y + 1) + ")";
+				if (hitObject.GetComponentInParent <RaycastText> () != null) {
+					RaycastText raycastText = hitObject.GetComponentInParent <RaycastText> ();
+					coordinatesText = raycastText.getRaycastInfoText ();
+				} else {
+					coordinatesText = "(" + Math.Round (hitObjectPos.x - offset.x) + ',';
```

Comment:
We should probably commit to using only the RaycastText object from the hitObject. When no RaycastText is found we should simply ignore the hit, and assume we want no text to show up.

Reply:
Oh I think I put that in temporarily to not break the code when I only had one prefab with the RaycastText script on it. I actually did mean to remove it.